docs/getting_started.html

```diff
@@ -0,0 +1,212 @@
+    <div class="col-4">
+      <div class="list-group" id="list-tab" role="tablist">
+        <a class="list-group-item list-group-item-action active" id="list-one-list" data-toggle="list" href="#list-one" role="tab" aria-controls="one">
+          1. Gather Dependencies
+        </a>
+        <a class="list-group-item list-group-item-action" id="list-two-list" data-toggle="list" href="#list-two" role="tab" aria-controls="two">
+          2. Create Project
+        </a>
+        <a class="list-group-item list-group-item-action" id="list-three-list" data-toggle="list" href="#list-three" role="tab" aria-controls="three">
+          3. Create/Run Terraform code
+        </a>
+        <a class="list-group-item list-group-item-action" id="list-four-list" data-toggle="list" href="#list-four" role="tab" aria-controls="four">
+          4. Create/Run Inspec Tests
+        </a>
+      </div>
+    </div>
+    <div class="col-8">
+      <div class="tab-content" id="nav-tabContent">
+        <div class="tab-pane fade show active" id="list-one" role="tabpanel" aria-labelledby="list-one-list">
+          Install Terraform: <a href="https://www.terraform.io/downloads.html" style="color: #32c850;">
+            https://www.terraform.io/downloads.html
+          </a>
+          <br>
+          Install Ruby: <a href="https://www.ruby-lang.org/en/documentation/installation/" style="color: #32c850;">
+            https://www.ruby-lang.org/en/documentation/installation/
+          </a>
+          <br><br>
+          Additional installation methods:
+          <br><br>
+          <div class="row">
+            <div class="col-sm">
+              Mac OS (homebrew):
+<div class="highlight"><pre class="syntax-highlight shell"><code>brew install terraform
+brew install ruby
+</code></pre></div>            </div>
+            <div class="col-sm">
+              Version managers:<br>
+              <a href="https://github.com/kamatama41/tfenv" style="color: #32c850;">tfenv</a>
+              <br>
+              <a href="https://github.com/rbenv/rbenv" style="color: #32c850;">rbenv</a>
+            </div>
+          </div>
+        </div>
+        <div class="tab-pane fade" id="list-two" role="tabpanel" aria-labelledby="list-two-list">
+          Create module and folder structure by running these commands:
+          <br><br>
+<div class="highlight"><pre class="syntax-highlight shell"><code>mkdir <span class="nt">-p</span> my_terraform_module <span class="o">&amp;&amp;</span> <span class="nb">cd </span>my_terraform_module
+
+mkdir <span class="nt">-p</span> <span class="nb">test</span>/integration/example/controls <span class="se">\</span>
+         <span class="nb">test</span>/fixtures/example/
+</code></pre></div>         Then install the necessary rubygems.
+         <br><br>
+<div class="highlight"><pre class="syntax-highlight shell"><code><span class="o">(</span>
+<span class="nb">cat</span> <span class="o">&lt;&lt;</span> <span class="no">EOP</span><span class="sh">
```

Comment:
We should be consistent in naming and displaying file contents in isolated, formatted blocks, rather than showing it inline like this.